adapt/intent.py

```diff
@@ -165,17 +171,22 @@ def validate_with_tags(self, tags, confidence):
             intent_confidence += confidence
 
         if len(self.at_least_one) > 0:
-            best_resolution = resolve_one_of(tags, self.at_least_one)
+            # we want to use a copy of the original tags here, as opposed to
+            # local_tags. This allows us to have overlaps between `required`
```

Comment:
should this be "as opposed to tags" instead of ...local_tags

Reply:
Yeah, this whole comment block was supposed to go. I'll update.